server/router.go

```diff
@@ -0,0 +1,10 @@
+package server
+
+import (
+	"github.com/ns1labs/orb/server/api/v1"
+)
+
+func (s *Server) RegisterRoutes() {
+	v1RouteGroup := s.engine.Group("/v1")
```

Comment:
We can add middleware to the v1 group when we have a little more clarity on what that looks like